src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/BrowserManagement.java

```diff
@@ -103,7 +103,7 @@
     protected Element element;
 
     @Autowired
-    private Robot robot;
+    private Robot robot = new Robot();
```

Comment:
Autowiring should instantiate robot -class, no explicit init needed.

Reply:
sorry, did not notice

Done